src/odemis/util/datacollector.py

```
@@ -0,0 +1,812 @@
+                else:
+                    logging.warning("DataArray not in valid format", exc_info=True)
+                    exporter = None
+
+                if exporter is not None:
+                    ext = exporter.EXTENSIONS[0]
+                    arc_name = f"{_sanitize_filename(key)}.{ext}"
+                    abs_path = tmp_dir / arc_name
+                    try:
+                        da = value if isinstance(value, model.DataArray) else model.DataArray(value)
+                        tiff.export(str(abs_path), da)
+                    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)